packages/cli-test/src/cli/shell.spec.ts

```diff
@@ -61,34 +61,104 @@ describe('shell module', () => {
     it('should invoke `assembleShellEnv` and pass as child_process.spawnSync `env` parameter', () => {
       const fakeEnv = { HEY: 'yo' };
       const assembleSpy = sandbox.stub(shell, 'assembleShellEnv').returns(fakeEnv);
-      const fakeCmd = 'echo "hi"';
-      shell.runCommandSync(fakeCmd);
+      const fakeCmd = 'echo';
+      const fakeArgs = ['"hi there"'];
+      shell.runCommandSync(fakeCmd, fakeArgs);
       sandbox.assert.calledOnce(assembleSpy);
-      sandbox.assert.calledWithMatch(runSpy, fakeCmd, sinon.match({ shell: true, env: fakeEnv }));
+      sandbox.assert.calledWithMatch(
+        runSpy,
+        sinon.match.string,
+        sinon.match.array,
+        sinon.match({ shell: true, env: fakeEnv }),
+      );
     });
     it('should raise bubble error details up', () => {
       runSpy.throws(new Error('this is bat country'));
       assert.throw(() => {
-        shell.runCommandSync('about to explode');
+        shell.runCommandSync('about to explode', []);
       }, /this is bat country/);
     });
+    if (process.platform === 'win32') {
```

Comment:
This fork in the test code is to test stuff on Windows vs. non-Windows properly, since we now include Windows-specific code.

This also explains why the code coverage dropped: since the code coverage is generated on a GitHub ubuntu runner, it won't exercise the Windows-specific code path, so the coverage reporter will report a lower coverage score. But, since our CI now runs tests on Windows, this code path _is_ exercised (you can see that in the GitHub Action output for the windows check [here](https://github.com/slackapi/node-slack-sdk/actions/runs/11635516867/job/32405055723#step:8:207))